app/src/main/java/app/wefridge/wefridge/placeholder/PlaceholderContent.kt

```diff
@@ -15,83 +13,10 @@ import java.util.concurrent.ThreadLocalRandom
 object PlaceholderContent {
 
```

Comment:
Any reason to not delete this file?

Reply:
Seems like we don't need placeholders anymore

Reply:
This file is still responsible for storing our items in an ArrayList locally. I thought about moving this Array into the ItemController.
As soon as we know how to handle the ParticipantItems without the PlaceholderContent, we can remove this file.